tests/backup/backup_helper.go

```diff
@@ -2063,6 +2063,36 @@ func CreateNamespaceLabelScheduleBackupWithoutCheck(scheduleName string, cluster
 	return resp, nil
 }
 
+// suspendBackupSchedule will suspend backup schedule
+func suspendBackupSchedule(backupScheduleName, backupScheduleUID, schPolicyName, schPolicyUID, OrgID string, ctx context.Context) error {
```

Comment:
Can you remove the input parameters `backupScheduleUID` and `schPolicyUID`
Get these values inside the helper function and we don't have to do it in the test case

Reply:
changed the code to fetch uid's within the functions.